functions/influxdb+grafana.bash

```diff
@@ -265,6 +265,7 @@ grafana_install(){
   if ! cond_redirect chmod -R 0750 /var/run/grafana; then echo "FAILED (chmod)"; return 1; fi
   if ! cond_redirect chown -R grafana:grafana /var/run/grafana; then echo "FAILED (chown)"; return 1; fi
   if ! zram_dependency install grafana; then return 1; fi
+  if ! mkdir -p /opt/zram/log.bind/grafana /var/log/grafana && chown grafana /opt/zram/log.bind/grafana /var/log/grafana; then echo "FAILED (create zram logdir)"; return 1; fi
```

Comment:
Might want to check if zram is installed before running as if it is not this is unnecessary. Please apply this feedback to all other instances.

Reply:
hmm, this will complicate the code do you insist ?

Reply:
Let me know how you think we should proceed. Merge as-is? If no, which missing changes do you consider mandatory ?

Reply:
I don't think that it will be an issue as the bug has to do with directories with special permissions not being correctly transferred to the lower directory. In a regular installation it won't hurt to skip the zram stuff if it is not enabled. I would go ahead and add a check that skips the permissions if zram is not installed. After that I think that is my only issue.

Once again, sorry for the delay on this I apparently missed your first comment on that one and never responded.